### PR TITLE
Early stages of EA event insertion

### PR DESCRIPTION
**Pull request by borkweb**
Other fixes encased herein:

* Fixing nonces
* Creating schedule records
* Marking imports as success/failure when one of those points are reached
* Credential form genericizing fixes

See: https://central.tri.be/issues/61161

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Don't leave debug_robot lying around | [New.php:130](https://github.com/moderntribe/the-events-calendar/blob/c7512e2565ead3e249606d1af84ab7c08f36b650/src/Tribe/Aggregator/Tabs/New.php#L130)
:x: | String &quot;finalizing&quot; does not require double quotes; use single quotes instead | [New.php:130](https://github.com/moderntribe/the-events-calendar/blob/c7512e2565ead3e249606d1af84ab7c08f36b650/src/Tribe/Aggregator/Tabs/New.php#L130)
:x: | Don't leave debug_robot lying around | [New.php:134](https://github.com/moderntribe/the-events-calendar/blob/c7512e2565ead3e249606d1af84ab7c08f36b650/src/Tribe/Aggregator/Tabs/New.php#L134)
:x: | String &quot;creating&quot; does not require double quotes; use single quotes instead | [New.php:134](https://github.com/moderntribe/the-events-calendar/blob/c7512e2565ead3e249606d1af84ab7c08f36b650/src/Tribe/Aggregator/Tabs/New.php#L134)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 1]
```diff
@@ -378,4 +439,197 @@ public function log_limit_reached_error() {
...
+
+		$results = array(
+			'updated' => 0,
+			'created' => 0,
+			'skipped' => 0,
+		);
+
+		$args = array();
+		$selected = array();
+    $has_row_selection = false;
```
Comment (bordoni) [inline src/Tribe/Aggregator/Record/Abstract.php]: Weird indent?

File: src/Tribe/Aggregator/Tabs/New.php [hunk 2]
```diff
@@ -158,11 +165,28 @@ public function handle_import_finalize( $data ) {
...
 		);
 
-		if ( 'csv' === $data['origin'] ) {
-			$result = $record->csv_insert_posts( $data );
+		$result = $record->insert_posts( $data );
+
+		if ( is_wp_error( $result ) ) {
+			$this->messages[ 'error' ][] = $result->get_error_message();
+
+			Tribe__Admin__Notices::instance()->register(
```
Comment (bordoni) [inline src/Tribe/Aggregator/Tabs/New.php]: Can we do it on a single line?
```php
Tribe__Admin__Notices::instance()->register( 'aggregator-import-failed', array( $this, 'render_notice_import_failed' ), 'type=error' )
```

I will add the `tribe_admin_notice()` public function.

**Comment by bordoni** [comment]
💙  👍

**Pull request merged**